nixos/modules/programs/vscode.nix

```
@@ -0,0 +1,61 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.vscode;
+in
+{
+  options.programs.vscode = {
+    enable = lib.mkEnableOption "VSCode editor";
+
+    defaultEditor = lib.mkEnableOption "" // {
+      description = ''
+        When enabled, configures Vscode to be the default editor
+        using the EDITOR environment variable.
+      '';
+    };
+
+    package = lib.mkPackageOption pkgs "vscode" { };
```

```suggestion
    package = lib.mkPackageOption pkgs "VSCode" {
      default = pkgs.vscode;
      extraDescription = "The final package will be customized with extensions from {option}`programs.vscode.extensions`";
    };
```
I think this should make it match casing in the documentation (not entirely sure). I also foresee users passing in `vscode-with-extensions` here without paying too much attention (I've done that myself before for some other modules not noticing there were options for them) so this extra description might be helpful

---

I'm going to leave the package as it is. I will add the extra desc though.